var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -39,6 +39,8 @@ class Hdf5(CMakePackage):
     version("develop-1.8", branch="hdf5_1_8")
 
     # Odd versions are considered experimental releases
+    # Note: These are still required to build some VOL adapters, but even releases should be
```

Comment:
Since 1.14.0 is out, 1.13.x versions should be removed as well as this comment.

Reply:
I think that is true, but I don't want to do it in the PR. It will require removing some of the VOL versions that need `hdf5@:1.13.2` as well and I would rather the VOL maintainers mad that change since they will know best which versions work with 1.14 and which do not. Once those are taken care of I think it is safe to  drop all of the 1.13.x versions.

Reply:
I think updating all VOL packages are right thing to do.